src/cli/search.rs

```diff
@@ -76,28 +75,21 @@ where
 
     let repos: Vec<RepoData> = repodata_query_func(specs).await.into_diagnostic()?;
 
-    let mut latest_packages: Vec<RepoDataRecord> = Vec::new();
-
-    for repo in repos {
-        // sort records by version, get the latest one of each package
-        let records_of_repo: HashMap<String, RepoDataRecord> = repo
-            .into_iter()
-            .sorted_by(|a, b| a.package_record.version.cmp(&b.package_record.version))
-            .map(|record| {
-                (
-                    record.package_record.name.as_normalized().to_string(),
-                    record.clone(),
-                )
-            })
-            .collect();
-
-        latest_packages.extend(records_of_repo.into_values().collect_vec());
-    }
+    let mut latest_packages: Vec<RepoDataRecord> = repos
+        .into_iter()
+        .flat_map(|repo| repo.into_iter().cloned().collect_vec())
+        .collect();
+
+    // sort all versions across all channels and platforms
+    latest_packages.sort_by(|a, b| a.package_record.version.cmp(&b.package_record.version));
+
+    // sort all versions across all channels and platforms
+    latest_packages.sort_by(|a, b| a.package_record.version.cmp(&b.package_record.version));
```

Comment:
```suggestion
    // sort all versions across all channels and platforms
    latest_packages.sort_by(|a, b| a.package_record.version.cmp(&b.package_record.version));
```
This is a duplicate right?

Reply:
wrongly merged